downstream/modules/platform/proc-install-aap-operator.adoc

```diff
@@ -1,14 +1,16 @@
-[id="proc-install-aap-operator{context}"]
-
-. Log in to {OCP}.
-. Navigate to *Operators* -> *Operator Hub*.
-. Locate the {PlatformName} operator and click on it.
-. Click *Install*.
-+
+[id="proc-install-aap-operator"]
 
 [NOTE]
 ====
 {OCP} clusters running on AWS do not support ReadWriteMany without adding NFS or other storage.
 ====
-. Upon successful installation of the platform operator, click on the *View operator* button on the installation confirmation message.
-.. You may also find the installed operator by navigating to *Operators* -> *Installed Operators*
+.Procedure
+. Log in to {OCP}.
+. Navigate to *Operators* -> *OperatorHub*.
+. Search for the {PlatformName} operator and click *Install*.
+. Select an *Installation Mode*, *Installed Namespace*, and *Approval Strategy*.
+. Click *Install*.
+
+The installation process will begin. When installation is complete a modal will appear notifying you that the {PlatformName} operator is installed in the specified namespace.
```

Comment:
suggestion based on preference: Add a comma after the word `complete` ->  `When installation is complete, a modal will appear...`